op-node/rollup/derive/span_channel_out.go

```diff
@@ -187,6 +219,10 @@ func (co *SpanChannelOut) AddSingularBatch(batch *SingularBatch, seqNum uint64)
 // it resets all the compression buffers because Span Batches aren't meant to be compressed incrementally.
 func (co *SpanChannelOut) compress() error {
 	co.compressed.Reset()
+	if co.compressionAlgo == BROTLI {
+		// reset but still need the version bit of 1
+		co.compressed.WriteByte(0b0001)
+	}
```

Comment:
I suggest a Span Channel Out function called `resetCompressed()` which does this BROTLI check and writes the byte for you. That way we would only have one place where this version bits are set.